string servoState
string trackState
float64 yaw
float64 pitch
